open-source-software-peer-review/intro.md

```diff
@@ -1,13 +1,127 @@
+   - [social media account](https://twitter.com/pyopensci).
+
+-   pyOpenSci packages that are in scope for the [Journal of Open-Source Software](https://joss.theoj.org/) and add the necessary accompanying short `paper.md` file, can, at the discretion of JOSS editors, benefit from a fast-tracked review process. <LINK TO MORE ON THIS>
+
+## Why do we need peer review for Python scientific software?
+
+pyOpenSci's [suite of packages](https://pyopensci.org/python-packages/) are fully 
+contributed by community members with a great diversity of skills. This diversity 
+of developer backgrounds results in a range of quality associated with the suite 
+of tools available to process scientific data.
+
+### Peer review helps maintain consistent open source software quality
+
+Peer review of python tools that support science is critical to enforcing 
+quality and usability standards. All pyOpenSci packages contributed by the 
+community undergo a transparent, constructive, non adversarial and open peer 
+review process. The goal of that process is to enforce commonly accepted standards.
+These standards include technical structure of the package, usability of the 
+package, documenting package functionality in a way that is accessible 
+to all levels of users and proper licensing and citation information. 
+
+### A truly community-founded review process
+
+Our peer review process is run by volunteer members of the Python scientific 
+community:
+
+* Editors manage the incoming package review submissions and ensure 
+reviews move forward progress of submissions; 
+* authors create, submit and improve their package; 
+* Reviewers, two per submission, examine the software code and user experience. 
+
+```{note}
+[This blog post](https://www.numfocus.org/blog/how-ropensci-uses-code-review-to-promote-reproducible-science/) written by editors from our partner organization, rOpenSci, is a good introduction to pyOpenSci software peer review 
 ```
+### How do I know a python package is pyOpenSci vetted?
+
+You can identify pyOpenSci packages that have been peer-reviewed by the green 
+"peer-reviewed" badge at the top their README, [![pyOpenSci](https://tinyurl.com/y22nb8up)]() linking to the specific issue
+where the tool was reviewed. [See this example from devicely, one of our packages](https://github.com/hpi-dhc/devicely).
 
+### How do reviews work?
 
+We use GitHub for our entire review process. We like GitHub because:
 
+* It's free to create an account
+* It facilitates collaboration and supports community around a package
+* It facilitates open discussion via issues
+* It supports version control
+* Numerous packages store their code bases on GitHub
+
+We make the most of [GitHub](https://github.com/)infrastructure in
+our review process.
+
+Each package review is contained within an issue in the [pyOpenSci/software-review GitHub repository](https://github.com/pyopensci/software-review/). 
+
+# TODO: ADD GOOD EXAMPLE HERE
```

Comment:
This would require adding an example from one of our reviews. it might be nice to use this ROS one for now because it's so clear and there is some really positive feedback from scott to maelle. Also kind of cool because maelle is staff now.